README.md

```diff
@@ -151,7 +152,9 @@ Add tags to your constructs. We recommend setting an `env` and `service` tag to
 
 ## How it works
 
-The Datadog CDK construct takes in a list of lambda functions and installs the Datadog Lambda Library by attaching the Lambda Layers for [Node.js][2] and [Python][1] to your functions. It redirects to a replacement handler that initializes the Lambda Library without any required code changes. Additional configurations added to the Datadog CDK construct will also translate into their respective environment variables under each lambda function (if applicable / required).
+The Datadog CDK construct takes in a list of lambda functions and installs the Datadog Lambda Library by attaching the Lambda Layers for [Node.js][2] and [Python][1] to your functions. It redirects to a replacement handler that initializes the Lambda Library without any required code changes. Additional configurations added to the Datadog CDK construct will also translate into their respective environment variables under each lambda function (if applicable / required). 
+
+While Lambda function based log groups are handled by the `addForwarder` function automatically, the construct has an additional function `addForwarderToLogGroups` which will subscribe the forwarder to any additional log groups of your choosing. 
```

Comment:
The only API's that a user can interact with are:
- addLambdaFunctions
- addForwarderToLogGroups
so I am wondering if it would make more sense to say: "While Lambda function based log groups are handled by the `addLambdaFunctions` function automatically,"  

since `addForwarder` is an internal function in which the user does not interact with.